package/network/config/firewall/files/firewall.config

```diff
@@ -33,6 +33,7 @@ config rule
 	option name		Allow-DHCP-Renew
 	option src		wan
 	option proto		udp
+	option src_port		67
```

Comment:
This seems reasonable.

Reply:
@KanjiMonster it looks like you're right about DHCPv6. I wonder if the same applies for DHCPv4 since the RFC says:
>DHCP uses UDP as its transport protocol.  DHCP messages from a client
>   to a server are sent to the 'DHCP server' port (67), and DHCP
>   messages from a server to a client are sent to the 'DHCP client' port
>   (68). A server with multiple network address (e.g., a multi-homed
>   host) MAY use any of its network addresses in outgoing DHCP messages.